buildenv/1.3.x/win32-static/harfbuzz-ng.build

```diff
@@ -1,27 +1,27 @@
 #!/usr/bin/env mumble-build
-# Copyright 2016 The 'mumble-releng' Authors. All rights reserved.
+# Copyright 2016-2018 The 'mumble-releng' Authors. All rights reserved.
 # Use of this source code is governed by a BSD-style license that
 # can be found in the LICENSE file in the source tree or at
 # <http://mumble.info/mumble-releng/LICENSE>.
 
 urls=(
-	"http://releng-distfiles.mumble.info/releng-distfiles/harfbuzz-1.4.5.tar.bz2"
-	"https://www.freedesktop.org/software/harfbuzz/release/harfbuzz-1.4.5.tar.bz2"
+	"https://s3.amazonaws.com/mumble-releng-distfiles/harfbuzz-1.8.8.tar.bz2"
+	"https://www.freedesktop.org/software/harfbuzz/release/harfbuzz-1.8.8.tar.bz2"
 )
 
 digests=(
-	"sha1:e979eb20b789c1fc47107ef93a584924e34dd195"
-	"sha256:d0e05438165884f21658154c709075feaf98c93ee5c694b951533ac425a9a711"
-	"blake2:8f2fb5c20c92afe3fb344bf4d4024090d68ab60c6d899c9805eec0bc6d0a714a3d98bfb6718ea4225da68b19e95db357c744dcba310076f4c56ed590985c9d1d"
+	"sha1:be496408b2bb3dc794891e8b0e977cd75f02d71d"
+	"sha256:a8e5c86e4d99e1cc9865ec1b8e9b05b98e413c2a885cd11f8e9bb9502dd3e3a9"
+	"blake2:c5b2b89f6fbefa004927c0ed5dc314d3f8fd4ce8c58918f4020a91247d9e781ff1ffb4ba05b138b4b1aebbdcad2041d749301143523a808f61f013e2efb5efdd"
 )
 
 function extract {
-	tar -jxf harfbuzz-1.4.5.tar.bz2
-	cd harfbuzz-1.4.5
+	tar -jxf harfbuzz-1.8.8.tar.bz2
+	cd harfbuzz-1.8.8
 }
 
 function prepare {
-	cp ${MUMBLE_BUILDENV_ROOT}/cmake/harfbuzz-ng-CMakeLists.txt ./CMakeLists.txt
```

Comment:
Does harfbuzz's own CMakeListsts.txt really do a static build? And against a shared CRT? :)